### ISSUE TITLE
signOut callbackUrl should support external url

### ISSUE THREAD

**Issue by tjhiggins** [Hebilicious/authjs-nuxt#136]
### Describe the feature

The current logic only supports internal routing after signOut.

My use case:
```ts
await signOut({ callbackUrl: 'https://tenant.auth0.com/oidc/logout?id_token_hint=blah&post_logout_redirect_uri=blah' })
```

Code to alter:
https://github.com/Hebilicious/authjs-nuxt/blob/3c5928aecaf02779b71341a10567260d5c11e772/packages/authjs-nuxt/src/runtime/lib/client.ts#L136

Easy workaround is to copy the signOut method and change the useRouter line to the following:
```ts
await navigateTo(url, { external: true });
```

### Additional information

- [ ] Would you be willing to help implement this feature?

**Comment by Hebilicious** [comment]
Hi @tjhiggins, afaik `useRouter().push` works with external URLs already, so this change wouldn't change anything. However, it's possible that the callback url is not being applied properly because of this https://github.com/Hebilicious/authjs-nuxt/blob/3c5928aecaf02779b71341a10567260d5c11e772/packages/authjs-nuxt/src/runtime/lib/client.ts#L134C16-L134C17

Could you confirm that this is what you'd expect?

**Comment by Hebilicious** [comment]
Got it. This should be a pretty straightforward change then.

**Comment by mrleblanc101** [comment]
Yes, this is an issue for Keycloak since clicking "signOut" only sign you out of the application, and not from the SSO.
So the next time you click "signIn", you are automatically logged back, without having to input your username/password.
![2024-01-15 14 39 52](https://github.com/Hebilicious/authjs-nuxt/assets/17392251/729dbf3b-ede2-43a5-968e-27a6a5f99d75)

**Comment by PeterPilley** [comment]
Hi is this resolved? I have the same issue as @mrleblanc101
